glfs.c

```diff
@@ -472,7 +473,7 @@ static glfs_t* tcmu_create_glfs_object(struct tcmu_device *dev,
 	}
 
 	ret = glfs_init(fs);
-	if (ret) {
+	if (!ret) {
```

Comment:
Was this some testing code that you meant to delete, or how has it worked before?

Reply:
This too.